introduction.md

```diff
@@ -1,2 +1,6 @@
 Hello!  Once upon a time I was doing the Mastering Github course.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?